src/pages/Home/index.js

```diff
@@ -0,0 +1,534 @@
+                    </Typography>
+                  </CardContent>
+                </Card>
+              </Grid>
+              <Grid item xs={12} ms={6} md={4}>
+                <Stack alignItems="center">
+                  <img src={require("./assets/down-swirly.svg")} />
+                </Stack>
+                <Card>
+                  <CardContent className={classes.cardConten}>
+                    <Typography align="center" gutterBottom>
+                      Become confident with our spoken English track
+                    </Typography>
+                  </CardContent>
+                </Card>
+              </Grid>
+            </Grid>
+          </Container>
+          {/* ********************** Section 5 ******************* */}
+          <Container maxWidth="sm">
+            <Stack alignItems="center">
+              <img src={require("./assets/down-swirly.svg")} />
+            </Stack>
+            <Typography sx={{ mt: 3 }} align="center" gutterBottom>
+              Our aim is to help you get technical and soft skills to become
+              ready for advanced learning schools such as Zoho or Navgurukul
+            </Typography>
+          </Container>
+
+          {/* ********************** Section 6 ******************* */}
+
+          <Container maxWidth="md" sx={{ mt: 10 }}>
+            <Typography
+              variant="h5"
+              component="h2"
+              align="center"
+              color="textPrimary"
+              gutterBottom
+            >
+              Our Partners
+            </Typography>
+            <Typography
+              sx={{ mt: 4 }}
+              variant="h2"
+              align="center"
+              color="primary"
+            >
+              30+
+            </Typography>
+            <Typography align="center" color="textPrimary" gutterBottom>
+              Partners Across India
+            </Typography>
+
+            <Grid sx={{ mt: 1 }} container spacing={2}>
+              <Grid item xs={2}>
+                <img
+                  className={classes.partner}
+                  src={require("./assets/Wipro.png")}
+                />
+              </Grid>
+              <Grid item xs={2}>
+                <img
+                  className={classes.partner}
+                  src={require("./assets/Infosys.png")}
+                />
+              </Grid>
+              <Grid item xs={2}>
+                <img
+                  className={classes.partner}
+                  src={require("./assets/Milaap.png")}
+                />
+              </Grid>
+              <Grid item xs={2}>
+                <img
+                  className={classes.partner}
+                  src={require("./assets/Askforhope.png")}
+                />
+              </Grid>
+              <Grid item xs={2}>
+                <img
+                  className={classes.partner}
+                  src={require("./assets/Infosys.png")}
```

Comment:
Remember to give images `alt` attributes for accessibility purposes.

Reply:
Ok